godot-core/src/builtin/array.rs

```diff
@@ -675,6 +675,26 @@ impl<T: GodotType> fmt::Debug for Array<T> {
     }
 }
 
+impl<T: GodotType + fmt::Display> fmt::Display for Array<T> {
+    /// Formats `Array` to match Godot's string representation
+    ///
+    /// Example:
+    /// ```no_run
+    /// # use godot::prelude::*;
+    /// let array = array![1,2,3,4];
+    /// assert_eq!(format!("{}", array), "[1, 2, 3, 4]");
```

Comment:
This misses closing code tags. "Example" could also be a section title like elsewhere.
I also made things a bit more compact and less ambiguous with "array" identifiers:
```suggestion
    /// # Example
    /// ```no_run
    /// # use godot::prelude::*;
    /// let a = array![1, 2, 3, 4];
    /// assert_eq!(format!("{a}"), "[1, 2, 3, 4]");
    /// ```
```